js/aes_gcm.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Subtle-based AES-GCM decryption supported on all browser types.
+ */
+
+/** 
+ * Possibly wrap an object in a Promise.
+ * @param {Object} op
+ * @return {!Promise}
+ */
+function wrapCryptoOp(op) {
+  if (typeof op.then == 'function') {
+    return op;
+  }
+  return new Promise(function (resolve, reject) {
```

Comment:
No space between "function" and "(". And use "=>" as much as possible.

Reply:
Done.